tutorials/terraform/data-transfer/data-transfer-mes-mos.tf

```
@@ -0,0 +1,161 @@
+# Infrastructure for the Managed Service for ElasticSearch, Managed Service for OpenSearch, and Data Transfer
+#
+# RU: https://cloud.yandex.ru/docs/data-transfer/tutorials/mes-to-mos
+# EN: https://cloud.yandex.com/en/docs/data-transfer/tutorials/mes-to-mos
+#
+# Specify the following settings:
+locals {
+  # Managed Service for ElasticSearch cluster settings:
+  create_mes        = 1 # Set to 0 to disable cluster creation if you use standalone ElasticSearch.
```

```suggestion
  create_mes        = 1  # Set to 0 to disable cluster creation if you use standalone ElasticSearch.
```

Давай выровгняем со строкой ниже

---

Выровнял все через `terraform fmt`